File: GullSharksLib/Models/Game.cs
namespace GullSharksLib;

public class Game
{
    public int ID { set; get; }
    public string GameName { set; get; }
    public int Asset_ID { set; get; }
    public int GameDetails_ID { set; get; }
    public double PriceInCAD { set; get; }

    public bool IsDeleted { get; set; }
}

File: GullSharksWeb/src/models/Game.ts
import { Asset } from "./Asset";
import { GameDetails } from "./GameDetails";
import { GameReview } from "./GameReview";

export interface Game {
  id: number;
  gameName: string;
  asset_ID: number;
  gameDetail_ID: number;
  priceInCAD: number;
  isDeleted: boolean;
  gameDetails?: GameDetails;
  gameAsset?: Asset;
  srcFront?: string;
  srcBack?: string;
  src?: string;
  rating?: number;
  textColor?: string;
  reviews?: GameReview[];
}
